content/codepipeline/role.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Create IAM Role"
+date: 2018-10-087T08:30:11-07:00
+weight: 10
+draft: true
+---
+
+In an AWS CodePipeline, we are going to use AWS CodeBuild to deploy a sample Kubernetes service.
+This requires an [AWS Identity and Access Management](https://aws.amazon.com/iam/) (IAM) role capable of interacting
+with the EKS cluster.
+
+In this step, we are going to create an IAM role and add an inline policy that we will use in the CodeBuild stage
+to interact with the EKS cluster via kubectl.
+
+Create the role:
+
+```
+ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)
+TRUST="{ \"Version\": \"2012-10-17\", \"Statement\": [ { \"Effect\": \"Allow\", \"Principal\": { \"AWS\": \"arn:aws:iam::$ACCOUNT_ID:root\" }, \"Action\": \"sts:AssumeRole\" } ] }"
```

Comment:
Let's put some carriage returns between the separate commands to increase readability.